app/src/main/java/org/oppia/app/recyclerview/OnItemDragListener.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.recyclerview
+
+import androidx.recyclerview.widget.RecyclerView
+
+/** Listener for handling action of  [DragItemTouchHelperCallback]. */
```

Comment:
Please fix the spacing here as well. In general, I suggest proofreading your PR ahead of time to try and catch these smaller inconsistencies.

Reply:
done and will keep this in mind for future before assigning any PR